src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+.searchForm {
+  position: relative;
+  margin-top: 20px;
+}
+
+.icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.smallIcon {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
```

Comment:
you can just add the padding to the whole input and remove top and left

Reply:
I used top and left for position, so I don’t understand why I have to add padding to the whole input, as my icon will be in the wrong place